src/renderer/src/components/editor/FolderEditor/FolderEditor.tsx

```
@@ -0,0 +1,148 @@
+    } else {
+      alert(`이름 변경 실패: ${result.message}`)
+    }
+  }
+
+  const dropdownData = seriesList.map((series) => ({
+    image: series.coverImagePath,
+    value: series.path,
+    label: series.title
+  }))
+
+  const hanldeNewFileCreateButton = async () => {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)